test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Reader/JsonLight/FeedAnnotationIntegrationTests.cs

```diff
@@ -534,9 +535,10 @@ private void CreateMessageReader(string payloadBody, bool forEntry, bool isRespo
             const string payloadSuffix = "}";
             string payload = payloadPrefix + payloadBody + payloadSuffix;
 
-            var container = ContainerBuilderHelper.BuildContainer(null);
+            var container = ServiceProviderHelper.BuildServiceProvider(null);

```

Comment:
Maybe a search and replace to find `var container = ` and replace it with `var serviceProvider = ` would be in order?